x/mongo/driver/topology/server_test.go

```diff
@@ -205,7 +205,7 @@ func TestServer(t *testing.T) {
 		s.connectionstate = connected
 		s.pool.connected = connected
 
-		wce := driver.WriteConcernError{"", 10107, "not master", []byte{}, []string{}}
+		wce := driver.WriteConcernError{"", 10107, "not master", []byte{}, []string{}, nil}
```

Comment:
nit: we've added enough fields that this is not really readable anymore. Can we expand this to a multi-line definition where the values are labeled with field names?